processor/deltatocumulativeprocessor/internal/delta/delta.go

```diff
@@ -65,3 +74,11 @@ type ErrOutOfOrder struct {
 func (e ErrOutOfOrder) Error() string {
 	return fmt.Sprintf("out of order: dropped sample from time=%s, because series is already at time=%s", e.Sample, e.Last)
 }
+
+type ErrGap struct {
+	From, To pcommon.Timestamp
+}
+
+func (e ErrGap) Error() string {
+	return fmt.Sprintf("gap in stream from %s to %s", e.From, e.To)
```

Comment:
As a user, what does it mean to me? Is there an action item, like "check that you don't have X", or "check that all Ys are ..." ?